src/app/bundles/bundle-details/bundle-details.component.ts

```diff
@@ -7,9 +7,16 @@ import { Component, OnInit } from '@angular/core';
 })
 export class BundleDetailsComponent implements OnInit {
 
+  name: string;
+  description: string;
+
   constructor() { }
 
   ngOnInit() {
   }
 
+  saveBundleDetails() {
```

Comment:
we might need to consider at least creating a dummy service that we invoke here. but might something for later